tests/6.0/map-type_modifier/test_map-type_modifier_default.c

```diff
@@ -20,11 +20,12 @@ int test_maptype_modifier_default() {
 
   #pragma omp target enter data map(to : a)
 
-  #pragma omp target map(present, from : a)
+  #pragma omp target map(present)
```

Comment:
You lost the variable name:
```suggestion
  #pragma omp target map(present : a)
```